src/api.js

```diff
@@ -23,3 +23,18 @@ export const getArticleById = article_id => {
     return res.data.article;
   });
 };
+
+export const updateVote = (article_id, inc_votes) => {
+  return newsApi
+    .patch(`articles/${article_id}`, { inc_votes })
+    .then(res => {
+      return res.data.article.votes;
+    })
+    .catch(err => Promise.reject(err));
+};
```

Comment:
Not sure what this promise reject is achieving? If we make it into a catch block, the promise has already been rejected

Reply:
catch block removed